bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableGrpcClient.java

```diff
@@ -135,7 +134,12 @@ public void onCompleted() {
   }
 
   /**
-   * Factory method to create a Cloud Bigtable data client.
+   * Create a new BigtableGrpcClient. Note, the caller needs to take responsibility of
```

Comment:
I think this comment should be in the constructor of BigtableGrpcClient, at least this is where it is for BigtableTableAdminGrpcClient. Also, it doesnt appear for BigtableClusterAdminGrpcClient